lib/pry-byebug/helpers/location.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module PryByebug
+  module Helpers
+    #
+    # Compatibility helper to handle source location
+    #
+    module Location
+      module_function
+
+      #
+      # Current file in the target binding. Used as the default breakpoint
+      # location.
+      #
+      def current_file(source = target)
+        return source.source_location[0] if source.respond_to?(:source_location)
```

Comment:
Here is could be a comment about Ruby versions, warnings.

Also, the `source_location` is since Ruby 1.9 or earlier, and `pry-byebug` has Ruby 2.3 as minimal, so… I think we're not needing in `eval("__FILE__")` at all.

If so — we can just replace the code, even without Helper, I guess.

Reply:
You're right ! Since `source_location` is available in Ruby's versions above minimal requirements, I agree, there is no need for eval or a helper at all

Reply:
We've faced with unavailability `Binding#source_location` in Ruby < 2.6. I think, here is still can be a comment about this.

Reply:
Added comment